app/views/quotes/_quote.html.erb

```diff
@@ -1,13 +1,22 @@
 <%= quote.quoted_person %> said, <q><%= quote.body %></q>, <%= quote.location %>.
 <% if quote.created_by == current_user %>
-  <%= link_to "delete quote", quote_path(quote), method: :delete  %>
-  <%= link_to "edit quote", edit_quote_path(quote) %>
+  <%= link_to "Delete quote", quote_path(quote), method: :delete  %>
+  <%= link_to "Edit quote", edit_quote_path(quote) %>
```

Comment:
fun fact, the paths can also be shortened to just:

```
link_to "Delete quote", quote, method: :delete
link_to "Edit quote", [:edit, quote]
```
If you haven't done so before, I highly recommend reading through all of: http://guides.rubyonrails.org/routing.html